sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -285,7 +300,9 @@ public <K, V> KafkaConsumer<K, V> createConsumer(
     KafkaConsumer<K, V> consumer =
         new KafkaConsumer<>(consumerProperties, keyDeSerializer, valueDeSerializer);
 
-    if (micrometerConsumerListener != null) {
+    if (micrometerConsumerListener == null) {
+      LOGGER.error("MicrometerConsumerListener is not initialized! Metrics will not be recorded.");
+    } else {
```

Comment:
I guess, with the initialization check, the `micrometerConsumerListener` can't be `null` here.